From e6180034998c63193f5d66c223b2eae65f208a08 Mon Sep 17 00:00:00 2001
From: Arend van Spriel <arend@broadcom.com>
Date: Wed, 25 Feb 2015 11:02:39 +0100
Subject: [PATCH] hwdb: add sdio identifiers for Broadcom WLAN cards

This patch adds the sdio identifiers known to be supported by
the brcmfmac open-source driver.

(cherry picked from commit 0c591b6c92ded66bea5ed615796b0f8c1cd6b422)
(cherry picked from commit c10e229f8222b92117ba38045ddb3e4d7951244a)
---
 hwdb/sdio.ids | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb/sdio.ids b/hwdb/sdio.ids
index 8a4c7136d2..d61729744e 100644
--- a/hwdb/sdio.ids
+++ b/hwdb/sdio.ids
@@ -34,6 +34,16 @@
 	5347  GDM72xx WiMAX
 02d0  Broadcom Corp.
 	044b  Nintendo Wii WLAN daughter card
+	a887  BCM43143 WLAN card
+	4324  BCM43241 WLAN card
+	4329  BCM4329 WLAN card
+	4330  BCM4330 WLAN card
+	4334  BCM4334 WLAN card
+	a94c  BCM43340 WLAN card
+	a94d  BCM43341 WLAN card
+	4335  BCM4335/BCM4339 WLAN card
+	a962  BCM43362 WLAN card
+	4354  BCM4354 WLAN card
 02db  SyChip Inc.
 	0002  Pegasus WLAN SDIO Card (6060SD)
 02df  Marvell Technology Group Ltd.
